### ISSUE TITLE
SystemUpdateAsync causes "Property ListId was not loaded"

### ISSUE THREAD

**Issue by jansenbe** [pnp/pnpcore#570]
### Discussed in https://github.com/pnp/pnpcore/discussions/569

<div type='discussions-op-text'>

<sup>Originally posted by **berntrune** October 11, 2021</sup>
I am trying to update Modifed field using PNP SystemUpdateAsync.
When using UpdateAsync, all normal item fields are stored successfully (except Modified of course).
When changing to SystemUpdateAsync I get "Property ListId was not yet loaded". (see stack trace below)

 using (var ctx = await _pnpContextFactory.CreateAsync(new Uri("http://site")))
                {
                    var doc = await ctx.Web.Lists.GetByTitleAsync("Documents");
                    var file = await doc.RootFolder.Files.AddAsync("file.docx", fileStream, true);
                    file.ListItemAllFields["Title"] = "new title";
                    file.ListItemAllFields["Modified"] = DateTime.Now;
                    //await file.ListItemAllFields.UpdateAsync();  
                    await file.ListItemAllFields.SystemUpdateAsync();
                }

   at PnP.Core.Model.TransientObject.GetValue[T](String propertyName)
   at PnP.Core.Model.SharePoint.File.get_ListId()
   at PnP.Core.Model.SharePoint.ListItem.<PrepareUpdateCall>d__80.MoveNext()
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at PnP.Core.Model.SharePoint.ListItem.<SystemUpdateAsync>d__83.MoveNext()
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at LargeFileTransfer.TransferFile.<Run>d__3.MoveNext() in C:\DevopsGit\Bikube.Azure.Services.BergenFunction\TransferFile.cs:line 109

Normally, in csom, I've been able to swap these methods without any issues.
Any clues?
</div>

**Comment by jansenbe** [comment]
@berntrune : doing some more digging I found a related bug. After adding a file one needs to load the ListItemAllFields property and that failed using `await file.LoadAsync(p => p.ListItemAllFields);` as approach (while the approach shown above worked). I've fixed this and with tomorrow's nightly build both approaches should work. I'll add a note to the docs to clarify that the need for the ListItemAllFields load after adding a file.
